docs/advanced-features/custom-error-page.md

```diff
@@ -54,25 +54,25 @@ If you want to render the built-in error page you can by importing the `Error` c
 
 ```jsx
 import Error from 'next/error'
-import fetch from 'isomorphic-unfetch'
+import fetch from 'node-fetch'
 
-const Page = ({ errorCode, stars }) => {
+export async function getServerSideProps() {
```

Comment:
This is incorrect, _error doesn't exactly support `getServerSideProps` or `getStaticProps` yet as it needs the `err` that is provided in general.

Reply:
@timneutkens This is for the example of a page that wants to import the error page and use it, not for the _error page itself.

Reply:
Ah I see! Missed that